relation_engine/taxa/ncbi/test/parsers_test.py

```diff
@@ -168,13 +178,247 @@ def test_node_provider_complex():
             'scientific_name':            "sciname74",
             'rank':                       "no rank",
             'strain':                     False,
+            'species_or_below':           False,
             'aliases':                    [],
             'ncbi_taxon_id':              74,
             'gencode':                    11,
         },
     ]
 
 
+_NON_SPECIES_RANKS = [
```

Comment:
Is there a reason this is here and not in the ranks.py file?

Reply:
I suppose it's just used by the tests but, idk

Reply:
Because if the tests just import from `ranks.py` rather than having their own source of truth and someone fatfingers `ranks.py`, the tests will pass when they shouldn't.